tcms/templates/case/get_details_case_run.html

```diff
@@ -12,7 +12,7 @@ <h4 style="padding-bottom:3px;">Comments</h4>
 				{% if perms.django_comments.add_comment %}
 				<form class="update_form" method="POST">
 					<table border="0" cellpadding="0" cellspacing="0" width="100%">
-						{% get_comment_form for execution as comment_form %}
+						<textarea name="comment" cols="40" rows="10" maxlength="3000" id="id_comment"></textarea>
 						<tr>
 							<td>
 								{{ comment_form.comment }}
```

Comment:
this is actually the text area input field. Because you've removed the assignment operation above the value below do not render and are now useless. Remove them.

Also remove `load comments` from the top of this template b/c it is no longer needed.